donut/modules/groups/utils/insert_permissions.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+from donut.pymysql_connection import make_db
+
+valid_permissions = {
+    "Devteam":
+    {
+        True: [1], # All
+        False: [1] # All
+    },
+    "IHC":
+    {
+        True: [2, 4, 5], #Edit Rotation Info, Editing Pages, Upload documents
+        False: [2, 4, 5] #Edit Rotation Info, Editing Pages, Upload documents
+    },
+    "ASCIT":
+    {   # Editing Pages, Upload documents, View Bodfeedback summary,
+        # Edit Bodfeedback, Edit Bodfeedback emails, View Bodfeedback emails
+        True: [4, 5, 8,
+            9, 10, 11,
+            # All the calendars
+            21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32,
+            # Surveys
+            33],
+        # View Bodfeedback summary, Edit Bodfeedback, View Bodfeedback emails
+        False: [8, 9, 11,
+            # Surveys
+            33]
+    },
+    "Academics and Research Committee (ARC)":
+    {
+        # View Arcfeedback summary, Edit Arcfeedback, Edit Arcfeedback emails
+        # View Arcfeedback emails, Surveys
+        True: [13, 14, 15, 16, 33],
+        # View Arcfeedback summary, Edit Arcfeedback, View Arcfeedback emails,
+        # Surveys
+        False: [13, 14, 16, 33]
+    },
+    "Avery":
+    {   # Edit Avery Calendar
+        True: [21],
+    },
+    "Blacker":
+    {
+        # Edit Blacker Calendar
+        True: [24],
+    },
+    "Dabney":
+    {
+        # Edit Dabney Calendar
+        True: [25],
+    },
+    "Fleming":
+    {
+        # Edit Fleming Calendar
+        True: [26],
+    },
+    "Lloyd":
+    {
+        # Edit Fleming Calendar
+        True: [27],
+    },
+    "Page":
+    {
+        # Edit Fleming Calendar
+        True: [28],
+    },
+    "Ricketts":
+    {
+        # Edit Ricketts Calendar
+        True: [29],
+    },
+    "Ruddock":
+    {
+        # Edit Ruddock Calendar
+        True: [30],
+    },
+    "Review Committee":
+    {
+        # Editing Pages, Upload documents, Surveys
+        True: [4, 5, 33]
+    }
+}
+
+
+def insert_permissions(env):
+    db = make_db(env)
+    try:
+        db.begin()
+        for group in valid_permissions:
```

Comment:
Might be a little cleaner to iterate over `group, group_permissions in valid_permissions.items()`. Ditto for `control, control_permissions in group_permissions.items()`